File: BaekJoon/BruteForce/2798.py
N, M = map(int ,input().split())

li = list(map(int, input().split()))

case = []

for i in range(0,len(li)-2):
    for j in range(i+1, len(li)-1):
        for k in range(j+1, len(li)):
            if li[i]+li[j]+li[k] <= M:
                case.append(li[i]+li[j]+li[k])
print(max(case))   

File: BaekJoonSwift/BaekJoonSwift/Bruth Force/2798.swift
import Foundation

let iValue = readLine()!.split(separator: " ").map { Int($0)! }
let len = iValue[0]
let goal = iValue[1]
var max = -1
let numbers = readLine()!.split(separator: " ").map { Int($0)! }

for i in 0..<len-2 {
    for j in i+1..<len-1 {
        for k in j+1..<len {
            let temp = numbers[i] + numbers[j] + numbers[k]
            if temp <= goal && temp > max {
                max = temp
            }
        }
    }
}
print(max)
